app/containers/Admin/ReportingYear/ReportingYearTable.tsx

```diff
@@ -17,10 +18,28 @@ function formatListViewDate(date: string) {
   return dateTimeFormat(date, 'days_string');
 }
 
-function isDatePast(date: string) {
-  const d = defaultMoment(date);
+function getMostRecentlyClosedYear(query) {
+  const orderedByCloseTimeDesc = query.allReportingYears.edges
+    .slice()
+    .sort((yearA, yearB) => {
+      const yearACloseTime = defaultMoment(yearA.node.applicationCloseTime);
+      const yearBCloseTime = defaultMoment(yearB.node.applicationCloseTime);
+
+      return yearACloseTime.isBefore(yearBCloseTime)
+        ? 1
+        : yearBCloseTime.isBefore(yearACloseTime)
+        ? -1
+        : 0;
```

Comment:
double ternary operators (a quinary operator? :upside_down_face: ) are a bit hard to read. Consider if/else or some other notation for it.